### ISSUE TITLE
Amazon - failed to create the EC2 instance

### ISSUE THREAD

**Issue by kataflok** [jlund/streisand#170]
Hi jlund,

Thanks for all you are doing here -- remarkable!

I've deleted all previous subscriptions and subscribed following the new link.

I also deleted the entire streisand folder and recloned the streisand repository to my instal machine -- which has already successfully installed to Digital Ocean.

It's still giving the below.

TASK: [genesis-amazon | Use the AMI ID for Oregon when the region is us-west-2] *** 
ok: [127.0.0.1]

TASK: [genesis-amazon | Create the EC2 instance] ****************************** 
failed: [127.0.0.1 -> 127.0.0.1] => {"failed": true}
msg: Instance creation failed => AuthFailure: Not authorized for images: [ami-17510927]

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
to retry, use: --limit @/Users/userfolder/amazon.retry

127.0.0.1 : ok=12 changed=5 unreachable=0 failed=1

Are there certain kinds of AWS accounts that lock out Streisand perhaps?

**Comment by df-sh** [comment]
The posted fix https://github.com/jlund/streisand/commit/ccf1633b85477e21dd2c7caa832cf4b59090a961

 wont work even when subscribing to the Debian in the setup.


If you have subscribed to https://aws.amazon.com/marketplace/pp/B00AA289LU from streisand setup:
open playbooks/roles/genesis-amazon/defaults/main.yml and replace whats in there with this:

```
aws_debian7_california_ami_id: "ami-da524a9f"
aws_debian7_ireland_ami_id: "ami-0355d974"
aws_debian7_oregon_ami_id: "ami-4b51097b"
aws_debian7_sao_paulo_ami_id: "ami-376ed22a"
aws_debian7_signapore_ami_id: "ami-e46d47b6"
aws_debian7_sydney_ami_id: "ami-33502409"
aws_debian7_tokyo_ami_id: "ami-a6ccd5a7"
aws_debian7_virginia_ami_id: "ami-38de9950"

aws_instance_type: "t1.micro"
```

Then you get to sit through the slow mirroring of tor and osx tunnelblick but it should work

**Comment by jlund** [comment]
I got the AMI IDs in ccf1633 from the list of official Debian images for Debian 7.8 on the AWS Marketplace. I verified that they worked before committing the change. Now it appears that Debian is only publishing official images for Debian 8.1 and 8.0, and all of these AMIs disappeared sometime within the past few days.

@df-sh: Where did you find these AMI IDs that are still working?

I am planning on spending some time this weekend to make Streisand work on Debian 8. That will become the new foundation. Then the official 8.1 images can be used and this problem will be solved. I would strongly prefer to use the official images in favor of any whose provenance is unknown.

**Comment by df-sh** [comment]
Following the link from streisand to Debian on amazon store:  https://aws.amazon.com/marketplace/pp/B00AA289LU


![awsdeb](https://cloud.githubusercontent.com/assets/12028761/8370682/f24e9dee-1bfd-11e5-826a-c29edc80f2c7.PNG)

After subscribing, press continue and then press the manual launch with ec2 console tab..

![amilist](https://cloud.githubusercontent.com/assets/12028761/8370691/1ec5d9c8-1bfe-11e5-9056-ebf5ea18de27.PNG)

**Comment by samchelsea009** [comment]
As a noob,.. I think in addition to the Debian version 7 vs 8 AMI issue. One problem with modifying the main.yml to an available image like ami-38de9950, is apparently that image is closed to new users per amazon.

I guess these issues are related though...

**Comment by df-sh** [comment]
Did you subscribe to the different Debian builds from the Amazon store?  My AWS account was three days old when I started using the AMI's I posted

**Comment by samchelsea009** [comment]
Sorry for the delay. I ended up getting things to work using GCE. Thanks
for your suggestion though.

**Comment by df-sh** [comment]
what did you do to get it to work?

**Comment by samchelsea009** [comment]
I switched to Google Cloud.

**Comment by samchelsea009** [comment]
Sorry, I'm not trying to be short here. I was unable to get Amazon EC2
working, so I tried Google Cloud and was able to get that working. Thanks
for your help, again. I'm not a coding buy but if I can help in any other
way, please let me know.

**Comment by df-sh** [comment]
No worries, its not my project, I just try to help out where I can.  I still havent gotten Google Cloud to work from my system yet due to CA problems... Any trouble in the Google deploy?

**Comment by jlund** [comment]
Ubuntu 14.04 is now the official base foundation for Streisand, and things should be working perfectly on AWS once again! All of the relevant commits have been merged to the master branch as part of issue #174 and you can start launching new Streisand instances today.

I am using the official AWS Ubuntu 14.04 images, so no marketplace steps are necessary.
